test/fixtures/site 1/views/with_preprocessor_error.hbs

```diff
@@ -1,6 +1,7 @@
 {{!
 {
+  "title": "Original title",
   "preprocessor": "error.js"
 }
 }}
-No error happened!
\ No newline at end of file
+No error happened!
```

Comment:
I test that the error page content is rendered instead (`Oh no (500)` or `Not here!`), in the new tests. Some older tests already used `No error happened!`, this is not a new line.